txpool/tx_object.go

```diff
@@ -81,13 +81,11 @@ func (o *txObject) Executable(chain *chain.Chain, state *state.State, headBlock
 		}
 	}
 
-	if o.BlockRef().Number() > headBlock.Number() {
+	// tx valid in the future
```

Comment:
// Tx is considered Executable when the BlockRef has passed in reference to the HeadBlock